docs/editor/github.md

```diff
@@ -35,7 +37,7 @@ When you do something that requires GitHub authentication, you'll see a prompt t
 
 ![Authentication Prompt](images/github/auth-prompt.png)
 
-Follow the steps to sign into GitHub and return to VS Code.
+Follow the steps to sign into GitHub and return to VS Code. If authenticating with an existing repository doesn’t work automatically, you may need to manually provide a personal access token. See [Creating a personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) for more information. 
```

Comment:
It would also be good to note that to enter the personal access token, the user should click on the "Signing into GitHub..." status bar item in VS Code that appears when they start the sign in process

Reply:
We could also note that for the GitHub Pull Requests and Issues extension, the personal access token should be created with scopes `read:user`, `user:email`, and `repo`. https://github.com/microsoft/vscode-pull-request-github/wiki#personal-access-token-authentication